topovis/ttk/readme.md

```diff
@@ -1,60 +1,28 @@
 # ttk Module
 
-Generate wrappers around all the ttkvtk filters by parsing the paraview xml files
-describing the filters.
+This module provides processors wrapping the [Topology ToolKit](https://topology-tool-kit.github.io/), short TTK. The processors are based on the XML information on the various TTK ParaView filters.
 
-to build with vcpkg add  ```"topology-tool-kit"```
-to the vcpkg.json file in the main inviwo repo
-that will install vtk and ttk.
-
-For setting stuff up in cmake is use this preset:
+The ttk module depends on the Inviwo VTK module (`misc/vtk`) as well as the TTK and VTK libraries. The VTK functionality is provided by the Inviwo module. To simplify things, we recommend the usage of vcpkg for both TTK and VTK using the following CMake preset.
```

Comment:
This is windows only

Reply:
might explain how to adjust for other systems

Reply:
Which would be to use the installed packages?

Reply:
Think I have used vcpkg on all platforms for vtk